Add date and ignored urls

SergeiLomako opened the pull request.

RomainLanz commented:
Hey @SergeiLomako! 👋 

Could you please add a bit of context on your PR and not simply dropping some code changes? 😄

SergeiLomako commented:
Hi @RomainLanz! Sorry for not adding a description. I added url with query string and datetime to the logger. Also in the config, added the ability (ignoredUrls (type Array)) to ignore some URLs in the form of additional settings

RomainLanz commented:
Seems that tests aren't passing. Could you please fix your code?
